www/js/controllers.js

```diff
@@ -73,6 +74,12 @@ angular.module('almond.controllers', [])
     $scope.long = $rootScope.userLong;
   }, true);
 
+  // destinationService
+
+  $scope.dest = destinationService;
```

Comment:
I would prefer to switch this back to using a wrapped function instead of storing a reference to destination service to the scope.  It seems like an anti-pattern to allow anything within the same scope to mutate the state of destination service, from any type of context, and debugging this type of pattern would get much more difficult if we started to allow other parts of the app to do the same.  This isn't enough to block the pull request, but I will go back and change this.